packages/lib/invite/service.ts

```diff
@@ -52,7 +61,9 @@ export const updateInvite = async (inviteId: string, data: TInviteUpdateInput):
   }
 };
 
-export const deleteInvite = async (inviteId: string): Promise<TInvite> => {
+export const deleteInvite = async (inviteId: string): Promise<TInvite | null> => {
```

Comment:
I don't think it can return `null`. Either the record is there and it gets returned or it throws. Regarding this maybe it's a good idea to catch the error ;-)

Reply:
Do you mean to catch the error in the place where this service is called?

Reply:
I mean in the case that the record that you want to delete doesn't exist. Like returning a RessourceNotFound error then.
We would need this in every service and I'm thinking if a prisma call error handler would make sense for the catch function:

```handlePrismaError(e);```

that does something like this:

```
if status code xx return RessourceNotFoundError,
else if error code xx return UniqueConstraintError
else return DatabaseError
```

What do you think?

Reply:
Yeah it sounds like a good idea, the current pattern is for a delete not to return null instead throw a RessourceNotFoundError, I can just follow that pattern now to be consistent.

I am considering not making this task longer by implementing a handle error for all catch in all services, if necessary I could tackle it in a separate PR 👍🏾